defi/src/api2/routes/index.ts

```diff
@@ -447,6 +447,12 @@ async function getDimensionsMetadataRoute(_req: HyperExpress.Request, res: Hyper
   return successResponse(res, await getDimensionsMetadata(), 60);
 }
 
+export function setProRoutes(router: HyperExpress.Router, _routerBasePath: string) {
+  router.get("/summaryBreakdown/:type/:name", ew(getDimensionBreakdownProtocolFileRoute))
```

Comment:
can you change this to /v2/financial-statement/:name, for now, lets restrict to aggregated monthly/quartly of combined fee data